trello/settings.py

```diff
@@ -0,0 +1,128 @@
+"""
+Django settings for trello project.
+
+Generated by 'django-admin startproject' using Django 3.2.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.2/ref/settings/
+"""
+
+from pathlib import Path
+import environ
+import os
+
+
+env = environ.Env(
```

Comment:
So what I should provide by DEBUG env value to turn off DEBUG mode in django ?

Reply:
To have DEBUG enabled, you must set it to one of boolean true strings ('yes', 'y', 'on', 'true, '1'). So any other value (or lack of the value) will result in default setting of DEBUG=False